vive_msgs/msg/ViveSystem.msg

```diff
@@ -0,0 +1,2 @@
+Controller[] controllers
```

Comment:
I suggest calling this file `Trackables.msg` or something.
It is already a `vive_msgs` so `Vive` saying again is redundant.

Reply:
Perhaps `vive_msgs` should be renamed to `openvr_msgs`, as nothing in the message type is vive-specific.